caffe2/CMakeLists.txt

```diff
@@ -1847,6 +1847,9 @@ if(BUILD_TEST)
         if(NOT MSVC)
               target_compile_options(${test_name}_${CPU_CAPABILITY} PRIVATE -Wno-ignored-qualifiers)
         endif(NOT MSVC)
+        if(NOT USE_SYSTEM_SLEEF)
```

Comment:
Not familiar with this but where is `USE_SYSTEM_SLEEF` defined?

Reply:
@jgong5 Aligned to here: https://github.com/pytorch/pytorch/blob/dd8a24b8b79bf1882ccad8c28f80b5438634b38c/aten/src/ATen/CMakeLists.txt#L435 . It control use sleef shared/static.